test/functionalTest/cases/1577_servicepath_get_subs/get_subs_with_servicepath.test

```diff
@@ -0,0 +1,307 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET Subscriptions V2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. Create 2 subscriptions for 2 servicepaths
+# 02. GET /v2/subscriptions for 1 servicepath
+# 03. GET /v2/subscriptions for /#
+# 04. GET /v2/subscriptions for inexistent servicepath
+#
+
+echo "01. Create 2 subscriptions for 2 servicepaths"
+echo "============================================="
+payload='{
+  "entities": [
+    {
+      "type": "Room",
+      "isPattern": "false",
+      "id": "ConferenceRoom"
+    },
+    {
+      "type": "Room",
+      "isPattern": "false",
+      "id": "OfficeRoom"
+    }
+  ],
+  "attributes": [
+    "temperature",
+    "occupancy",
+    "lightstatus"
+  ],
+  "reference": "http://127.0.0.1:'${LISTENER_PORT}'/notify",
+  "duration": "P5Y",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+      "condValues": [
+        "temperature",
+        "timestamp"
+      ]
+    }
+  ],
+  "throttling": "PT5S"
+}'
+orionCurl --url /v1/subscribeContext --payload "${payload}" --json  --tenant VALENCIA --servicePath /Valencia/basuras
```

Comment:
We can remove `--tenant VALENCIA` in all the steps in this tests: filtering by service/tenant is already working. In fact, CB is ignoring this in this test, as SHELL-INIT is no starting it with `-multitentant`.